tests/test_homepage.py

```diff
@@ -0,0 +1,9 @@
+def test_home_page(test_client):
+    """
+    GIVEN a Flask application
+    WHEN the '/' page is requested (GET)
+    THEN check the response is valid
+    """
+    response = test_client.get('/')
+    assert response.status_code == 200
+    assert b"MusicXML to ABC Notation Converter" in response.data
```

Comment:
Why is there a "b" in front of the string here?

Reply:
to note that it is a binary string, i could get rid of that

Reply:
Thanks for the clarification. Not a big deal for now. Let's first get the build to pass